src/main/resources/templates/signUp.html

```diff
@@ -1,28 +1,59 @@
 <!DOCTYPE html>
-<html xmlns:th="http://www.thymeleaf.org" lang="en">
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="title" content="Create User">
-    <link th:href="@{/css/bootstrap.min.css}" rel="stylesheet">
-    <link th:href="@{/css/fontawesome/fontawesome-all.min.css}" rel="stylesheet">
-    <link th:href="@{/css/main.css}" rel="stylesheet" type="text/css">
-    <title>Create User</title>
+    <title>Title</title>
 </head>
 <body>
-<div class="container-fluid">
-    <div class="row">
-        <div class="container">
-            <div class="row">
-                <div class="col-sm-4 sign-up--form order-first order-lg-last">
-                    <div class="row">
-                          <!--Form should be here-->
-                    </div>
-                </div>
-            </div>
+    <form style="    display: flex;
+    flex-direction: column;
```

Comment:
Don't add styles as attribute into the tag. It would be better, if you create separate css file, and declare the new class there.